src/tools/vs-code.md

```diff
@@ -41,56 +41,65 @@ There are a couple ways to create a new project.
 To create a new Flutter project from the Flutter
 starter app template:
 
- 1. Open the Command Palette
-    (`Ctrl`+`Shift`+`P` (`Cmd`+`Shift`+`P` on macOS)).
- 1. Select the **Flutter: New Project** command and press `Enter`.
- 1. Select **Application** and press `Enter`.
- 1. Select a **Project location**.
- 1. Enter your desired **Project name**.
+1. Open the Command Palette.
+
+   - On Windows or Linux,
+     press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>.
+   - On macOS,
+     press <kbd>Cmd</kbd> + <kbd>Shift</kbd>+<kbd>P<kbd>.
+
+1. Tyoe `flutter`.
+1. Select the **Flutter: New Project**.
+1. Press <kbd>Enter</kbd>.
+1. Select **Application**.
+1. Press <kbd>Enter</kbd>.
+1. Select a **Project location**.
+1. Enter your desired **Project name**.
 
 ### Opening a project from existing source code
 
 To open an existing Flutter project:
 
- 1. Click **File > Open** from the main IDE window.
- 1. Browse to the directory holding your existing
-    Flutter source code files.
- 1. Click **Open**.
+1. From the main IDE window, go to **File** > **Open**.
+1. Browse to the directory holding your existing
+   Flutter source code files.
+1. Click **Open**.
 
 ## Editing code and viewing issues
 
 The Flutter extension performs code analysis that
 enables the following:
 
-* Syntax highlighting
-* Code completions based on rich type analysis
-* Navigating to type declarations
-  (**Go to Definition** or `F12`),
+- Syntax highlighting
+- Code completions based on rich type analysis
+- Navigating to type declarations
+  (**Go to Definition** or press <kbd>F12</kbd>),
```

Comment:
The "Go to Definition" doesn't have an action. Either add one for that or drop it from the key binding.